uktrade/data-hub-frontend
# Add new contact to form in contact loop

## Description of change
Fixes a bug where adding a new contact as part of adding a referral or interaction was meant to add the new contact as preselected on the form. 

## Test instructions

- Go to the 'Companies' list, and pick a company
- Choose either 'add a referral' or 'record an interaction'
- While completing form, follow link to 'add a new contact'
- Add a new contact
- On returning to the referral or interaction form, that new contact should be preselected.

### After

_Add a screenshot_

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
